packages/xo-web/src/common/xo-line-chart/index.js

```diff
@@ -604,7 +605,11 @@ export const IopsLineChart = injectIntl(
             nValues: length,
             endTimestamp,
             interval,
-            valueTransform: value => `${value.toPrecision(3)} /s`,
+            valueTransform: value =>
+              humanFormat(value, {
+                decimals: 3,
+                unit: ' /s',
```

Comment:
So this could show something like `4.2 k /s`. Is that what we want? Why not `4.2 kiops`?